apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff

```diff
@@ -543,9 +573,72 @@
       </trans-unit>
-      <trans-unit id="Database export &amp; import" xml:space="preserve">
-        <source>Database export &amp; import</source>
-        <target>Экспорт и импорт архива чата</target>
+      <trans-unit id="Database encrypted!" xml:space="preserve">
+        <source>Database encrypted!</source>
+        <target>База данных зашифрована!</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Database encryption passphrase will be updated and stored in the keychain.&#10;" xml:space="preserve">
+        <source>Database encryption passphrase will be updated and stored in the keychain.
+</source>
+        <target>Пароль базы данных будет изменен и сохранен в Keychain.
+</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Database encryption passphrase will be updated.&#10;" xml:space="preserve">
+        <source>Database encryption passphrase will be updated.
+</source>
+        <target>Пароль базы данных будет изменен.
+</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Database error" xml:space="preserve">
+        <source>Database error</source>
+        <target>Ошибка базы данных</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Database is encrypted using a random passphrase, you can change it." xml:space="preserve">
+        <source>Database is encrypted using a random passphrase, you can change it.</source>
+        <target>База данных зашифрована случайным паролем, вы можете его поменять.</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Database is encrypted using a random passphrase. Please change it before exporting." xml:space="preserve">
+        <source>Database is encrypted using a random passphrase. Please change it before exporting.</source>
+        <target>База данных зашифрована случайным паролем. Пожалуйста, поменяйте его перед экспортом.</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Database passphrase" xml:space="preserve">
+        <source>Database passphrase</source>
+        <target>Пароль базы данных</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Database passphrase &amp; export" xml:space="preserve">
+        <source>Database passphrase &amp; export</source>
+        <target>Пароль и экспорт базы</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Database passphrase is different from saved in the keychain." xml:space="preserve">
+        <source>Database passphrase is different from saved in the keychain.</source>
+        <target>Пароль базы данных отличается от пароля сохрененного в Keychain.</target>
```

Comment:
same typo was in android - just in case suggestion wasn't applied yet
```suggestion
        <target>Пароль базы данных отличается от пароля сохраненного в Keychain.</target>
```